### PR TITLE
some new translations completed

### PR DESCRIPTION
**Pull request by jschwender**

### CODE REVIEW

File: locale/cups_de.po [hunk 1]
```diff
@@ -3045,16 +3045,16 @@ msgid "Envelope C7"
...
-msgstr ""
+msgstr "Umschlag Choukei 3"
 
 msgid "Envelope Choukei 3 Long Edge"
-msgstr ""
+msgstr "Umschlag Choukei 3 lange Seite"
 
 msgid "Envelope Choukei 4"
-msgstr ""
+msgstr "Umschalg Choukei 4"
```
Comment (IOOI-SqAR) [inline locale/cups_de.po]: Typo: should be "Umschlag" not "Umschalg"

File: locale/cups_de.po [hunk 2]
```diff
@@ -3063,94 +3063,94 @@ msgid "Envelope Feed"
...
 msgid "Envelope Invite"
-msgstr ""
+msgstr "Umschlag Invite"
 
 msgid "Envelope Italian"
 msgstr "Umschlag italienisch"
 
 msgid "Envelope Kaku2"
-msgstr ""
+msgstr "Umschalg Kaku2"
```
Comment (IOOI-SqAR) [inline locale/cups_de.po]: Typo: should be "Umschlag" not "Umschalg"

File: locale/cups_de.po [hunk 3]
```diff
@@ -3063,94 +3063,94 @@ msgid "Envelope Feed"
...
-msgstr ""
+msgstr "Umschlag Monatch"
 
 msgid "Envelope PRC1"
-msgstr ""
+msgstr "Umschlag PRC1"
 
 msgid "Envelope PRC1 Long Edge"
-msgstr ""
+msgstr "Umschalg PRC1 lange Seite"
```
Comment (IOOI-SqAR) [inline locale/cups_de.po]: Typo: should be "Umschlag" not "Umschalg"

File: locale/cups_de.po [hunk 4]
```diff
@@ -3063,94 +3063,94 @@ msgid "Envelope Feed"
...
 
 msgid "Envelope PRC10"
-msgstr ""
+msgstr "Umschalg PRC10"
```
Comment (IOOI-SqAR) [inline locale/cups_de.po]: Typo: should be "Umschlag" not "Umschalg"

File: locale/cups_de.po [hunk 5]
```diff
@@ -3063,94 +3063,94 @@ msgid "Envelope Feed"
...
 
 msgid "Envelope PRC10 Long Edge"
-msgstr ""
+msgstr "Umschalg PRC10 lange Seite"
```
Comment (IOOI-SqAR) [inline locale/cups_de.po]: Typo: should be "Umschlag" not "Umschalg"

File: locale/cups_de.po [hunk 6]
```diff
@@ -3063,94 +3063,94 @@ msgid "Envelope Feed"
...
 
 msgid "Envelope PRC2"
-msgstr ""
+msgstr "Umschalg PRC2"
```
Comment (IOOI-SqAR) [inline locale/cups_de.po]: Typo: should be "Umschlag" not "Umschalg"

File: locale/cups_de.po [hunk 7]
```diff
@@ -3063,94 +3063,94 @@ msgid "Envelope Feed"
...
 
 msgid "Envelope PRC2 Long Edge"
-msgstr ""
+msgstr "Umschalg PRC2 lange Seite"
```
Comment (IOOI-SqAR) [inline locale/cups_de.po]: Typo: should be "Umschlag" not "Umschalg"

File: locale/cups_de.po [hunk 8]
```diff
@@ -3063,94 +3063,94 @@ msgid "Envelope Feed"
...
 
 msgid "Envelope PRC3"
-msgstr ""
+msgstr "Umschalg PRC3"
```
Comment (IOOI-SqAR) [inline locale/cups_de.po]: Typo: should be "Umschlag" not "Umschalg"

File: locale/cups_de.po [hunk 9]
```diff
@@ -3063,94 +3063,94 @@ msgid "Envelope Feed"
...
 
 msgid "Envelope PRC3 Long Edge"
-msgstr ""
+msgstr "Umschalg PRC3 lange Seite"
```
Comment (IOOI-SqAR) [inline locale/cups_de.po]: Typo: should be "Umschlag" not "Umschalg"

File: locale/cups_de.po [hunk 10]
```diff
@@ -3063,94 +3063,94 @@ msgid "Envelope Feed"
...
-msgstr ""
+msgstr "Umschlag PRC4"
 
 msgid "Envelope PRC4 Long Edge"
-msgstr ""
+msgstr "Umschalg PRC4 lange Seite"
 
 msgid "Envelope PRC5 Long Edge"
-msgstr ""
+msgstr "Umschalg PRC5 lange Seite"
```
Comment (IOOI-SqAR) [inline locale/cups_de.po]: Typo: should be "Umschlag" not "Umschalg"

File: locale/cups_de.po [hunk 11]
```diff
@@ -3063,94 +3063,94 @@ msgid "Envelope Feed"
...
-msgstr ""
+msgstr "Umschlag PRC9 lange Seite"
 
 msgid "Envelope Personal"
-msgstr ""
+msgstr "Umschlag Personal"
 
 msgid "Envelope You4"
-msgstr ""
+msgstr "Umschalg You4"
```
Comment (IOOI-SqAR) [inline locale/cups_de.po]: Typo: should be "Umschlag" not "Umschalg"

File: locale/cups_de.po [hunk 12]
```diff
@@ -3063,94 +3063,94 @@ msgid "Envelope Feed"
...
 
 msgid "Envelope You4 Long Edge"
-msgstr ""
+msgstr "Umschalg You4 lange Seite"
```
Comment (IOOI-SqAR) [inline locale/cups_de.po]: Typo: should be "Umschlag" not "Umschalg"

File: locale/cups_de.po [hunk 13]
```diff
@@ -3063,94 +3063,94 @@ msgid "Envelope Feed"
...
-msgstr ""
+msgstr "Umschalg PRC3 lange Seite"
 
 msgid "Envelope PRC4"
-msgstr ""
+msgstr "Umschlag PRC4"
 
 msgid "Envelope PRC4 Long Edge"
-msgstr ""
+msgstr "Umschalg PRC4 lange Seite"
```
Comment (IOOI-SqAR) [inline locale/cups_de.po]: Typo: should be "Umschlag" not "Umschalg"

File: locale/cups_de.po [hunk 14]
```diff
@@ -3063,94 +3063,94 @@ msgid "Envelope Feed"
...
-msgstr ""
+msgstr "Umschlag Kaku3"
 
 msgid "Envelope Kaku3 Long Edge"
-msgstr ""
+msgstr "Umschlag Kaku3 lange Seite"
 
 msgid "Envelope Monarch"
-msgstr ""
+msgstr "Umschlag Monatch"
```
Comment (IOOI-SqAR) [inline locale/cups_de.po]: Typo: should be "Monarch" not "Monatch"

File: locale/cups_de.po [hunk 15]
```diff
@@ -3171,10 +3171,10 @@ msgid "Error: need hostname after \"-h\" option."
 msgstr "Fehler: Hostname ist nach der \"-h\" Option erforderlich."
 
 msgid "European Fanfold"
-msgstr ""
+msgstr "Europäisches Endospapier"
```
Comment (IOOI-SqAR) [inline locale/cups_de.po]: Typo: should be "Endlospapier" not "Endospapier"

File: locale/cups_de.po [hunk 16]
```diff
@@ -3171,10 +3171,10 @@ msgid "Error: need hostname after \"-h\" option."
...
 
 msgid "European Fanfold Legal"
-msgstr ""
+msgstr "Europäisches Endospapier Legal"
```
Comment (IOOI-SqAR) [inline locale/cups_de.po]: Typo: should be "Endlospapier" not "Endospapier"

File: locale/cups_de.po [hunk 17]
```diff
@@ -3315,22 +3315,22 @@ msgid "IPP attribute is not a member of the message."
...
-msgstr ""
+msgstr "IPP endCollection Wert nicht 0 Byte."
 
 msgid "IPP enum value not 4 bytes."
-msgstr ""
+msgstr "IPP enum Wert nicht 4 Byte."
 
 msgid "IPP extension tag larger than 0x7FFFFFFF."
-msgstr ""
+msgstr "IPP Erweiterung grösser als 0x7FFFFFFF."
```
Comment (IOOI-SqAR) [inline locale/cups_de.po]: should be "größer" not "grösser" if it is not considered Swiss german

File: locale/cups_de.po [hunk 18]
```diff
@@ -3354,22 +3354,22 @@ msgid "IPP name larger than 32767 bytes."
 msgstr "IPP Name länger als 32767 Byte"
 
 msgid "IPP nameWithLanguage value less than minimum 4 bytes."
-msgstr ""
+msgstr "IPP nameWithLanguage Wert weniger als mindestens 4 Byte."
 
 msgid "IPP octetString length too large."
-msgstr ""
+msgstr "IPP Oktett-Zeichenkettenlänge zu gross."
```
Comment (IOOI-SqAR) [inline locale/cups_de.po]: should be "groß" not "gross" if this is not considered to be Swiss german

File: locale/cups_de.po [hunk 19]
```diff
@@ -3715,10 +3715,10 @@ msgid "Never"
 msgstr "Nie"
 
 msgid "New credentials are not valid for name."
-msgstr ""
+msgstr "Neue Anmeldedaten sind für den Namen ungültig."
 
 msgid "New credentials are older than stored credentials."
-msgstr ""
+msgstr "Neue Anmeldedaten sind läter als die gespeicherten Anmeldedaten"
```
Comment (IOOI-SqAR) [inline locale/cups_de.po]: Typo: should be "älter" not "läter"

File: locale/cups_de.po [hunk 20]
```diff
@@ -4248,10 +4248,10 @@ msgid "Tabloid"
 msgstr "US Tabloid"
 
 msgid "Tabloid Oversize"
-msgstr ""
+msgstr "US Tabloid übergross"
```
Comment (IOOI-SqAR) [inline locale/cups_de.po]: should be "übergroß" not "übergross" if this is not considered to be Swiss german

File: locale/cups_de.po [hunk 21]
```diff
@@ -4248,10 +4248,10 @@ msgid "Tabloid"
...
 
 msgid "Tabloid Oversize Long Edge"
-msgstr ""
+msgstr "US Tabloid übergross lange Seite"
```
Comment (IOOI-SqAR) [inline locale/cups_de.po]: should be "übergroß" not "übergross" if this is not considered to be Swiss german

File: locale/cups_de.po [hunk 22]
```diff
@@ -4443,22 +4443,22 @@ msgid "US Legal"
 msgstr "US Lang"
 
 msgid "US Legal Oversize"
-msgstr ""
+msgstr "US Legal übergross"
```
Comment (IOOI-SqAR) [inline locale/cups_de.po]: should be "übergroß" not "übergross" if this is not considered to be Swiss german

File: locale/cups_de.po [hunk 23]
```diff
@@ -4443,22 +4443,22 @@ msgid "US Legal"
...
 msgid "US Letter"
 msgstr "US Letter"
 
 msgid "US Letter Long Edge"
-msgstr ""
+msgstr "US Letter lange Seite"
 
 msgid "US Letter Oversize"
-msgstr ""
+msgstr "US Letter übergross"
```
Comment (IOOI-SqAR) [inline locale/cups_de.po]: should be "übergroß" not "übergross" if this is not considered to be Swiss german

File: locale/cups_de.po [hunk 24]
```diff
@@ -4443,22 +4443,22 @@ msgid "US Legal"
...
 
 msgid "US Letter Oversize Long Edge"
-msgstr ""
+msgstr "US Letter übergross, lange Seite"
```
Comment (IOOI-SqAR) [inline locale/cups_de.po]: should be "übergroß" not "übergross" if this is not considered to be Swiss german

**Comment by michaelrsweet** [comment]
@IOOI-SqAR Thanks for the review, I'll apply those changes shortly.

**Comment by michaelrsweet** [comment]
[master 739ad3f3e] Typo changes after review (Issue #5264)

**Pull request merged**